.docs/content/1.installation/1.linux/1.installation.md

```diff
@@ -0,0 +1,164 @@
+
+## Setup environment variables
+
+Before running the script, you must set environment variables.
+
+From the root of the repository:
+
+```bash
+cd infrastructure/quick-deploy/localhost
+```
+
+Then, you can set environment variables in the file `envvars.sh`:
+
+```bash
+source envvars.sh
+```
+
+::alert{type="info"}
+Please, read the file `envvars.sh` before running it.
+::
+
+
+## Deploy
+
+Create data directory:
+
+```bash
+mkdir -p "${ARMONIK_SHARED_HOST_PATH}"
+```
+
+::alert{type="danger"}
+If you don't create the data directory, the deployment will failed when the compute plane will try to mount the shared directory.
+::
+
+::alert{type="info"}
+You understand why your pod don't start using `kubectl describe pod <pod_name> -n armonik`.
+::
+
+Finally, you can run the script to deploy ArmoniK on your local machine:
+
+```bash
+make deploy-all
+```
+
+Now, you take a coffee and wait for the end of the deployment. :coffee:
+
+You will easily recognize the end of the deployment when you see the following message:
+
+```hsl
+armonik = {
+  "admin_api_url" = "http://<your_ip>:5000/api"
+  "admin_app_url" = "http://<your_ip>:5000/admin"
+  "admin_old_url" = "http://<your_ip>:5000/old-admin"
+  "control_plane_url" = "http://<your_ip>:5001"
+  "grafana_url" = "http://<your_ip>:5000/grafana/"
+  "seq_web_url" = "http://<your_ip>:5000/seq/"
+}
+
+```
+
+You can also deploy each resource independently:
+
+```bash
+make deploy-keda
+```
+
+```bash
+make deploy-metrics-server
+```
+
+```bash
+make deploy-storage
+```
+
+```bash
+make deploy-monitoring
+```
+
+```bash
+make deploy-armonik
+```
+
+::alert{type="info"}
+The order matters because top level resources depend on the bottom level resources.
+::
+
+## Destroy
+
+To destroy all the infrastructure, you can run the following command:
+
+```bash
+make destroy-all
+```
+
+::alert{type="warning"}
+To reinstall, you must restart from [setup environment variables](#setup-environment-variables).
+::
+
+You can also destroy each resource independently:
+
+```bash
+make destroy-armonik
+```
+
+```bash
+make destroy-monitoring
+```
+
+```bash
+make destroy-storage
+```
+
+```bash
+make destroy-metrics-server
+```
+
+```bash
+make destroy-keda
+```
+
+::alert{type="info"}
+The order matters. You must destroy the resources in the reverse order of the deployment.
+::
+
+## Clean
+
+To clean all the generated files from the deployment, you can run the following command:
+
+```bash
+make clean-all
+```
+
+::alert{type="warning"}
+To reinstall, you must restart from [setup environment variables](#setup-environment-variables).
+::
+
+You can also clean each resource independently:
+
+```bash
+make clean-armonik
+```
+
+```bash
+make clean-monitoring
+```
+
+```bash
+make clean-storage
+```
+
+```bash
+make clean-metrics-server
+```
+
+```bash
+make clean-keda
+```
+
+::alert{type="info"}
+The order matters. You must clean the resources in the reverse order of the deployment.
+::
```

Comment:
Would it be useful to talk about the all-in-one deployment ? @lemaitre-aneo

Reply:
Yes, a section about this all in one could be a great idea! (currently, I don't know what does it mean)